Packs/CortexXDR/ReleaseNotes/6_1_24.md

```diff
@@ -0,0 +1,10 @@
+
+#### Playbooks
+
+##### Cortex XDR incident handling v3
+
+Updated the conditional task that determines whether to continue with the investigation and response.
```

Comment:
if found unhandled alerts... something like that

Reply:
changed to "Updated the conditional task that determines whether to continue with the investigation and response to any unhandled alerts if they are found."